src/shared/firewall-util-nft.c

```diff
@@ -891,10 +891,13 @@ static int nft_message_append_setelem_ip6range(
         return sd_netlink_message_close_container(m); /* NFTA_SET_ELEM_LIST_ELEMENTS */
 }
 
-static int fw_nftables_add_masquerade_internal(
+static int nft_set_element_op_in_addr(
```

Comment:
cryptic name...

Reply:
it appears to me the old, specifically named function should be retained as a minimal wrapper around the new function that just fills in the three relevant fields and then hands off to the generic calls

Reply:
I had `nft_set_element_add_in_addr()` `nft_set_element_del_in_addr()` before I merged them to this version. But yes, the name could be improved.

OK for retaining the old name.

Reply:
Now `_modified_` instead of `_op_`